website/docs/seo.mdx

```diff
@@ -218,3 +218,96 @@ Docusaurus uses your file names as links, but you can always change that using s
 Search engines rely on the HTML markup such as `<h2>`, `<table>`, etc., to understand the structure of your webpage. When Docusaurus renders your pages, semantic markup, e.g. `<aside>`, `<nav>`, `<main>`, are used to divide the different sections of the page, helping the search engine to locate parts like sidebar, navbar, and the main page content.
 
 Most [CommonMark](https://spec.commonmark.org/0.30/#atx-headings) syntaxes have their corresponding HTML tags. By using Markdown consistently in your project, you will make it easier for search engines to understand your page content.
+
+## Internationalization (i18n) SEO {#i18n-seo}
+
+When your Docusaurus site supports multiple languages, it's crucial to implement proper SEO practices to help search engines understand and serve the right content to users. Here's how to optimize your multilingual site:
+
+### Language Tags {#language-tags}
+
+Docusaurus automatically adds the appropriate `lang` attribute to your HTML:
+
+```html
+<html lang="en">
+  <!-- English content -->
+</html>
+
+<html lang="es">
+  <!-- Spanish content -->
+</html>
+```
+
+### Hreflang Tags {#hreflang-tags}
+
+Hreflang tags help search engines serve the correct language version to users. Docusaurus automatically generates these tags for all your language versions:
+
+```html
+<!-- For your English pages -->
```

Comment:
There's no information below that indicates that it's an "English page", so including this confuses readers.